charts/app-config/values-integration.yaml

```diff
@@ -564,6 +564,53 @@ govukApplications:
           secretKeyRef:
             name: signon-token-frontend-publishing-api
             key: bearer_token
+- name: draft-frontend
+  repoName: frontend
+  helmValues:
+    sentry:
+      createSecret: false
+    uploadAssets:
+      enabled: false
+    extraEnv:
+      - name: GOVUK_NOTIFY_TEMPLATE_ID
+        value: *frontend-notify-template
+      - name: PLEK_HOSTNAME_PREFIX
+        value: draft-
+      - name: SECRET_KEY_BASE
+        valueFrom:
+          secretKeyRef:
+            name: rails-secret-key-base
+            key: SECRET_KEY_BASE
+      - name: PUBLISHING_API_BEARER_TOKEN
+        valueFrom:
+          secretKeyRef:
+            name: signon-token-frontend-publishing-api
```

Comment:
should this not be "signon-token-draft-frontend-publishing-api"

like wise for other bearer tokens

Reply:
No, most of the draft apps use the same Doorkeeper apps and API users in Signon, except for draft-content-store and draft-router-api — see https://signon.integration.publishing.service.gov.uk/doorkeeper_applications and https://signon.integration.publishing.service.gov.uk/api_users.

Reply:
Thanks for checking though!

Reply:
ok thanks - good to know

Reply:
Oh, I just noticed `PUBLISHING_API_BEARER_TOKEN` is duplicated though (also in the non-draft one above that I copied from) 😅